### ISSUE TITLE
Long list of bookmarks jumps to top of page when adjusting/organizing bookmarks

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#33971]
## Description 

When you have a large list of bookmarks (such that you have to scroll down to view them all), if you:

- Delete a bookmark from the long list, the page will automatically jump to the top from where you are in the list
- Move a bookmark by dragging it from the list into another folder, the page jumps to the top of the list

This can be frustrating for users trying to organize a long list of bookmarks, since jumping to the top of the page every time causes them to lose their place in the list. Behavior should be that after each/any action, users position in the list/on the page should stay the same.

## Steps to Reproduce

You must have a long list of bookmarks — long enough force you to scroll to view them all.
   1. `Settings --> Bookmarks`
   2. Scroll down a long list of bookmarks and right-click to delete one of them
   3. Observe page jumps to top of list
   4. Repeat process for dragging a bookmark from long list to a new folder

## Reproduces how often: 

Every time

## Brave version (brave://version info)

v1.59.124

## Version/Channel Information:

- Can you reproduce this issue with the current release? Y
- Can you reproduce this issue with the beta channel? Y
- Can you reproduce this issue with the nightly channel? Y

## Miscellaneous Information:

User reported this and also reported that this behavior will happen if you navigate away from the Bookmark manager page. However, I am unable to reproduce this particular part of the issue:
https://community.brave.com/t/bookmarks-manager-menu-automatically-goes-to-top-when-deleting-a-bookmark-or-when-leaving-the-page/510220/3

**Comment by Brave-Matt** [comment]
+1 from Community — very similar issue here:
https://community.brave.com/t/bug-in-brave-browser-desktop-dragging-or-deleting-any-bookmarks-anywhere-within-brave-bookmarks-brings-it-top-not-staying-where-it-is/512335/8

**Comment by KRivva** [comment]
Will there be a fix soon?

**Comment by HomeStarRunnerTron** [comment]
WOW, it might be too soon to post this, but it looks like I can FINALLY delete or move bookmarks without automatically making the page scroll back to the top. I'm currently stresstesting, and I'm VERY happy with how consistently it stays in place. Just letting anyone else know who's been having to find workarounds for all of these months.

I'm currently on Version 1.68.128 Chromium: 127.0.6533.73 (Official Build) (arm64), on macOS. Don't know if this is the exact version that fixed things or not.

**2026 EDIT:** After a while of the reliability of Bookmarks slipping, this most recent update with the overhaul seems to have made a huge difference. Version 1.88.127 Chromium: 146.0.7680.71 (Official Build) (arm64). Just keep the Bookmarks in a different window, or it WILL start jumping to the top of the page again.

**Comment by lordparvis** [comment]
do you people have your sync chain on ?

**Comment by HomeStarRunnerTron** [comment]
> do you people have your sync chain on ?

No, I don't! Should I? In this current year, I'm finding that I can't scroll for too long without ending up back at some previous point in the list.

**Comment by Brave-Matt** [comment]
+1 https://www.reddit.com/r/brave_browser/comments/1uz828f/comment/oykvtyb/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

**Comment by Brave-Matt** [comment]
This is still an issue. @rebron can we get someone to take a look?
https://www.reddit.com/r/brave_browser/comments/1v77nwj/annoying_bookmark_glitch/
